docs/user-guide/new-ui/ui-improvements.md

```diff
@@ -11,49 +11,90 @@ status: new
-
-- Dates automatically display in your preferred regional format without manual configuration.
-- Improved readability and intuitive date interpretation for international teams.
-- Consistent data integrity and standardized storage across the entire system.
-- Seamless experience for users in different geographical locations.
-
-**Example - Table with Regional Date Display:**
+**Example Etendo Classic UI**
 
 <figure markdown="span">
     ![Etendo Classic Date Format](../../assets/user-guide/newui/classic-date.png){ align=left }
-    <figcaption>**Etendo Classic:** Dates in Etendo Classic display in the standard format configured in the system.
+    <figcaption>**Etendo Classic UI :** Dates in Etendo Classic UI display in the standard format configured in the system.
     </figcaption>
 </figure>
 
-
-**Example - New UI with Regional Configuration (en-US):**
-
+**Example Etendo UI with Regional Configuration (en-US):**
 
 <figure markdown="span">
     ![New UI Date Format with Regional Configuration](../../assets/user-guide/newui/newui-date.png)
-    <figcaption>**Etendo Main UI:** In the Etendo Main UI the same table displays dates automatically formatted according to the browser's regional settings. In this example `en-US` (United States) is applied, so dates appear in `MM/DD/YYYY` format. Users in other regions will see their corresponding regional formats, providing a localized and intuitive experience.</figcaption>
+    <figcaption>**Etendo UI:** In the Etendo UI the same table displays dates automatically formatted. In this example `en-US` (United States) is applied, so dates appear in `MM/DD/YYYY` format. </figcaption>
 </figure>
 
+###  Attachment Management
+
+The **Etendo UI** introduces a completely redesigned attachment management system that provides a more agile, visual, and comprehensive way to handle file attachments, both from **form view** and directly from **grid records**.
+
+#### Attachments Section in Form View
+
+The attachments section has been redesigned to be more intuitive. Files are now displayed as tags or chips, allowing for quick identification.
+
+![Attachment Section Expanded](../../assets/user-guide/newui/attachment-section-expanded.png)
+
+**File Upload**
+
+![Upload Modal Preview](../../assets/user-guide/newui/upload-modal-preview.png){align=right width=300}
+
+There are tree main ways to upload files within a record:
+
+1. **Drag & Drop:** Drag one or multiple files directly from your computer to the dotted **Drop Zone** in the attachments section.
+
+    !!! note
+        When dropping the file, a confirmation modal will open where you can verify the selected file and add an optional description before uploading it.
+
+2. **File Explorer:** Click on the upload zone (or upload icon) to open your operating system's file selector.
+
+3. **Import from URL:** The option to attach external resources via a link is included using the **Import from URL** button (*functionality subject to configuration*).
```

Comment:
*functionality subject to configuration* 

sujeta a que configuración ? esta implementado? En caso de estarlo debería documentarse, y sino deshabilitarlo